### PR TITLE
Created a standard logo component for the top bar

### PR DESCRIPTION
**Pull request by ronniebeggs**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Imported a higher-resolution V4V logo png
- Created a standard top bar logo component
- Added the logo to the scanning screen

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
Run `npm start` and navigate to the scanning screen. Check if the logo renders correctly and has higher resolution.

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
We should add this logo to all screens across the app. We should create a standardized top bar. 

## Relevant Links
This react-native-svg library was difficult to use for no reason: https://www.npmjs.com/package/react-native-svg

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🥦 CC: @sauhardjain

### CODE REVIEW

File: src/components/common/StandardLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styled from 'styled-components/native';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const v4vLogo = require('../../../assets/top-bar-logo.png');
```
Comment (sauhardjain) [inline src/components/common/StandardLogo.tsx]: We can remove the eslint issue by declaring the png type in our TypeScript types declarations. This can be done by adding `declare module '*.png';` to our `env.d.ts` file. 

Once you do this, you can simply import the logo as we would normally.

**Review by sauhardjain — CHANGES_REQUESTED** [review]
What a beautiful PR @ronniebeggs and great job on your first sprint! For next steps before merging this:
- [x] Rebase onto main using `git pull origin main -r`. This will pull any new changes from main and place all of your commits on the branch on top of the new changes. Make sure to resolve any merge conflicts before moving forward.
- [x] Resolve the comment about the logo import.

Reach out if you have any questions!

**Review by sauhardjain — APPROVED** [review]
